QUARKUS-3290.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-3290 - Generate source zip which contains the source code of the built artifacts
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-3290
+
+As part of discussions abound QUARKUS-2701, it was decided to generate source zip which contains the source code of the artifact from the internal Gerrit repository for upcoming 3.x release
+
+One of the reasons is to fully comply with https://issues.redhat.com/secure/attachment/12796750/screenshot-1.png from Red Hat compliance training
```

Comment:
Do we have this text in writing somewhere? Link to jira attachment with text isn't the best solution

Reply:
No text available that could be linked.

I'm not sure I should paste full trext into the test plan, that's why I linked the JIRA attachment.

```
Complete Corresponding Source Code
Complete Corresponding Source Code must include whatever is reasonably necessary for a package to be rebuilt
from source by a technically-skilled recipient with access to an appropriate build environment, without engaging in
excessive effort or experimentation. Items such as Makefiles, build and install scripts, build instructions, RPM spec
files, and POM files must be included with the source code as appropriate. Note that there are separate
requirements to make source code publicly available for Red Hat products for purposes of meeting our obligations
under the US and other export laws that may exceed or differ from the requirements described here. Please review
this link for more information on these separate requirements relating to export laws: separate requirements.
```